digitalpalireader/content/js/bv.js

```diff
@@ -1,7 +1,16 @@
 function bv(rnd,static) {
   var xmlhttp = new window.XMLHttpRequest();
-  xmlhttp.open("GET", 'chrome://digitalpalireader/content/etc/dbv.html', false);
+
+  let url = 'chrome://digitalpalireader/content/etc/dbv.html';
+  if (DPR_PAL.isWeb) {
```

Comment:
we support only the web. so as cleanup, you may remove the xul code paths - this cleanup can be done along with your next work item.